rwengine/src/engine/SaveGame.cpp

```diff
@@ -539,7 +539,11 @@ bool SaveGame::loadGame(GameState& state, const std::string& file)
 	auto icv = iconv_open("UTF-8", "UTF-16");
 	char* saveName = (char*)state.basic.saveName;
 
+#if defined(RW_NETBSD)
+	iconv(icv, (const char**)&saveName, &bytes, &outCur, &outSize);
```

Comment:
nit: avoid C style casts, other instances too

Also: what's different here, except for the cast? Wouldn't this only cause a warning?
What's the signature in NetBSD?